configs/records.config.default.in

```diff
@@ -71,6 +71,7 @@ CONFIG proxy.config.http.down_server.abort_threshold INT 10
 ##############################################################################
 CONFIG proxy.config.http.negative_caching_enabled INT 0
 CONFIG proxy.config.http.negative_caching_lifetime INT 1800
+CONFIG proxy.config.http.negative_caching_list STRING 204 305 403 404 405 414 500 501 502 503 504
```

Comment:
No - it's a policy goal to reduce the amount of things in this file. The defaults should be handled if needed in `RecordsConfig.cc`, as you've already done.

Reply:
But users need to put `204 305 403 404 405 414 500 501 502 503 504` in here on their own, and modify it if they want.

Reply:
Well, if you want, you can allow a special token like "default" to mean "add all the default status codes". Therefore to add 301, it would be "default 301".